mirrorsync.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
One more comment:
This is fine if this script will only be run on RockyLinux or even if it will only ever be run in an EL distribution.  But if there is a possibility that it may be run on another linux, or BSD, or ??? then the shebang would be more portable like this:
`#!/bin/env bash`